style.css

```diff
@@ -72,3 +91,158 @@ nav {
   display: inline-block;
   margin-right: 18px;
 }
+
+.works {
+  display: grid;
+  margin: 100px auto;
+  gap: 50px;
+}
```

Comment:
Please consider changing the value of the `display` property for each project to have a value of `display:flex` instead of `display:grid` as mentioned in the project requirements shown in the screenshot below :point_down: 
![flexbox](https://user-images.githubusercontent.com/16392046/191824720-9cef0cd1-fe8b-496c-a79a-2ff848c28a70.png)